packages/x-charts/src/PieChart/PieChart.tsx

```diff
@@ -135,6 +140,16 @@ const PieChart = React.forwardRef(function PieChart(inProps: PieChartProps, ref)
         slots={slots}
         slotProps={slotProps}
       />
+      <PiePlot slots={slots} slotProps={slotProps} onItemClick={onItemClick} />
+      <ChartsOverlay loading={loading} slots={slots} slotProps={slotProps} />
+      {!hideLegend && (
+        <ChartsLegend
+          direction="column"
+          position={{ vertical: 'middle', horizontal: isRtl ? 'left' : 'right' }}
+          slots={slots}
+          slotProps={slotProps}
+        />
+      )}
```

Comment:
This is just adding to the file instead of changing a single line 😅

Reply:
Oopsi, bad cherry pick resolution